ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -2318,6 +2318,12 @@ sflow:
 
 ```yaml
 snmp_server:
+  engine_ids:
+    local: < engine_name in hex >
+    remotes:
+      - id: < engine_name in hex >
+        address: <  hostname_or_ip_of_remote_engine >
```

Comment:
address: <  hostname | ip_of_remote_engine >

Reply:
I don't agree. Hostname of what. Normally we use the pipe to describe specific options. Like fixed strings. Here it is just a string.